packages/calypso-products/src/constants/types.ts

```diff
@@ -43,4 +41,7 @@ export const TYPES_LIST = < const >[
 	TYPE_PRO,
 	TYPE_STARTER,
 	TYPE_GOLDEN_TOKEN,
+	TYPE_JETPACK_STARTER,
+	TYPE_SECURITY_T1,
+	TYPE_SECURITY_T2,
```

Comment:
We did not have these Jetpack plan types added to the list for some reason. I went ahead and added them.